src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -443,8 +446,7 @@ void StartImportingBasicBlock(BasicBlock basicBlock)
             if (basicBlock.FilterStart || basicBlock.HandlerStart)
             {
                 Debug.Assert(basicBlock.EntryStack == null);
-
-                // TODO: Remember the kind of handler the block is for instead
```

Comment:
Could you please delete the for-loop and replace it with values cached on the basic block?